lib/internal/abort_controller.js

```diff
@@ -23,6 +23,9 @@ const {
   customInspectSymbol,
 } = require('internal/util');
 const { inspect } = require('internal/util/inspect');
+const {
+  ERR_INVALID_THIS
+} = require('internal/errors').codes;
```

Comment:
```suggestion
const {
  codes: {
    ERR_INVALID_THIS,
  }
} = require('internal/errors');
```

Reply:
Ah, I see. I copied this from [`lib/internal/encoding.js`](https://github.com/nodejs/node/blob/b461a1b1259b0b12f6fdbc45851e8b0546ddde84/lib/internal/encoding.js#L18-L24), but it looks like most other files use your suggested pattern.

Reply:
Yeah, I've been slowly trying to move them over :-)